examples/quickstart-pytorch/client.py

```diff
@@ -79,8 +96,7 @@ def load_data():
 
 # Load model and data (simple CNN, CIFAR-10)
 net = Net().to(DEVICE)
-trainloader, testloader = load_data()
-
+trainloader, testloader = load_data(standalone_mode=False)
```

Comment:
the `standalone_mode=False` is needed to return anything from the `click` function